src/services/user/model/user.schema.js

```diff
@@ -42,7 +42,7 @@ const userSchema = new Schema({
 
 	children: [{ type: Schema.Types.ObjectId, ref: 'user' }],
 	parents: [{ type: Schema.Types.ObjectId, ref: 'user' }],
-
+	defaultLanguage: { type: String },
```

Comment:
shouldnt this be the actual "language" instead of a "defaultLanguage"?

Reply:
done ;)